src/Integrations/Integrations/Roadrunner/RoadrunnerIntegration.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace DDTrace\Integrations\Roadrunner;
+
+use DDTrace\SpanData;
+use DDTrace\Integrations\Integration;
+use DDTrace\Type;
+use DDTrace\Util\Normalizer;
+
+/**
+ * Roadrunner integration
+ */
+class RoadrunnerIntegration extends Integration
+{
+    const NAME = 'roadrunner';
+
+    /**
+     * @return string The integration name.
+     */
+    public function getName()
+    {
+        return self::NAME;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function requiresExplicitTraceAnalyticsEnabling()
+    {
+        return false;
+    }
+
+    /**
+     * @return int
+     */
+    public function init()
+    {
+        if (!self::shouldLoad(self::NAME)) {
+            return Integration::NOT_LOADED;
+        }
+
+        $integration = $this;
+
+        ini_set("datadog.trace.auto_flush_enabled", 1);
+        ini_set("datadog.trace.generate_root_span", 0);
```

Comment:
Won't it already be too late for `generate_root_span` to be set?

Reply:
No, we have a handler on the DD_TRACE_GENERATE_ROOT_SPAN config, which removes an existing root span if that's the top-most span.